### ISSUE TITLE
Cross-category decoding

### ISSUE THREAD

**Issue by danielbrianbennett** [DDTBOX/DDTBOX#16]
Just a heads-up: I have been playing around with the cross-category decoding, and there appears to be a bug of some kind which inflates prediction accuracies significantly (by a mean of ~20% in a dataset of 23 participants). I spent some time going through the code this morning, but have not yet been able to work out where this comes from. It's something to do with how trials are taken and sorted.

**Comment by danielbrianbennett** [comment]
OK, I've worked out where this is coming from, and am working toward implementing a fix. Basically, the way the script currently does cross-category decoding, there are a couple of different ways that trials from the training set for one category can incidentally find themselves in the test set for the other category, thereby presumably inflating the apparent prediction accuracy. Will keep you updated on this.

**Comment by danielbrianbennett** [comment]
OK, there is now a pull request for code that should clear this up.
